tor/core/user_interaction.py

```diff
@@ -256,6 +257,16 @@ def process_done(
                 f" successful."
             )
             message = done_messages["completed_transcript"]
+            
+            transcription_count = blossom_user["gamma"]
+            is_promoted = check_promotion(transcription_count)
```

Comment:
I think we have to input `transcription_count + 1` here. The transcription count in the `blossom_user` object has been retrieved *before* the new transcription has been processed, so it will be one behind at this point.

Reply:
Yes Sir I understood the logic